crates/client/rpc-core/src/lib.rs

```diff
@@ -27,9 +27,26 @@ use starknet_core::types::{
 #[derive(Serialize, Deserialize)]
 pub struct Felt(#[serde_as(as = "UfeHex")] pub FieldElement);
 
+#[derive(Serialize, Deserialize)]
+pub struct PredeployedAccount {
+    pub contract_address: FieldElement,
+    pub contract_class: ContractClass,
+    pub balance: FieldElement,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct PredeployedAccountsInfo {
+    pub accounts: Vec<PredeployedAccount>,
+    pub private_key: FieldElement,
```

Comment:
private_key does not have to be a FieldElement, it can be anything. So I would have it be a `Vec<u8>`

Reply:
alright